serving/samples/README.md

```diff
@@ -7,7 +7,7 @@ more about Knative Serving resources.
 
 | Name                       | Description                                                                                                                                                                                                              |                                                                                                                                                                                                                            Languages                                                                                                                                                                                                                             |
 | -------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
-| Hello World                | A quick introduction that highlights how to deploy an app using Knative Serving.                                                                                                                                         | [C#](helloworld-csharp/README.md), [Clojure](helloworld-clojure/README.md), [Eclipse Vert.x](helloworld-vertx/README.md), [Haskell](helloworld-haskell/README.md), [Go](helloworld-go/README.md), [Java](helloworld-java/README.md), [Kotlin](helloworld-kotlin/README.md), [Node.js](helloworld-nodejs/README.md), [PHP](helloworld-php/README.md), [Python](helloworld-python/README.md), [Ruby](helloworld-ruby/README.md), [Rust](helloworld-rust/README.md) |
+| Hello World                | A quick introduction that highlights how to deploy an app using Knative Serving.                                                                                                                                         | [C#](helloworld-csharp/README.md), [Go](helloworld-go/README.md), [Java](helloworld-java/README.md), [Kotlin](helloworld-kotlin/README.md), [Node.js](helloworld-nodejs/README.md), [PHP](helloworld-php/README.md), [Python](helloworld-python/README.md), [Ruby](helloworld-ruby/README.md) |
```

Comment:
Should the Kotlin sample also be moved into the community folder?

Reply:
Kotlin is on the rise and Google is starting to support it, so I thought I'd keep it.

Reply:
SGTM, then!